### DISCUSSION TITLE
when setting gtk-titlebar = false and gtk-adwaita = true, no headbar is shown

### DISCUSSION THREAD

**Discussion by ttys3** [ghostty-org/ghostty#4101] [Feature Requests, Ideas] [answered]
when setting gtk-titlebar = false and gtk-adwaita = true, no headbar is shown

the issue was introduced by https://github.com/ghostty-org/ghostty/commit/a7c93cdfb16b61137ce086f3b6eed9bf7b94d73e

**Comment by jcollie** [comment, marked as answer]
This is working as expected. `titlebar` and `headerbar` are synonymous. It's rather unfortunate that the terminology is used interchangeably but that's what we have to deal with.
